src/Webform.js

```diff
@@ -1294,6 +1295,16 @@ export default class Webform extends NestedComponent {
       return console.warn('You should add a URL to this button.');
     }
   }
+
+  triggerRecaptcha() {
+    const recaptchaComponent = this.root.components.find((component) => {
```

Comment:
This will only search in the "root" scope of the form. You should use ```getComponent``` instead which is a method exposed to the nested components.

Reply:
@travist Thanks! Fixed it by using ```everyComponent``` since I need more complex search than just by key (which is what ```getComponent``` does as I see)